packages/ui/components/core/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import { Container, Title, Grid, Text, Stack, Group, createStyles } from '@mantine/core'
+
+import { Button } from './Button'
+import { SocialMediaIconButton } from './SocialMediaIconButton'
+import { BodyGrid } from '../layout'
+
+const useStyles = createStyles((theme) => ({
+	footer: {
+		backgroundColor: theme.other.colors.primary.footer,
+		['& a']: {
+			color: theme.colors.dark,
+		},
+	},
+}))
+
+const supportLinks = [
+	['Suggest an organization', '#'],
+	['Share feedback', '#'],
+	['Vetting process', '#'],
+	['Privacy statement', '#'],
+	['Anti-hate commitment', '#'],
+	['Digital accessibility', '#'],
+	['Disclaimer', '#'],
+]
+
+const connectLinks = [
+	['Inreach.org', '#'],
+	['Download the app', '#'],
+	['Subscribe to our newsletter', '#'],
+]
+
+const makeLinks = (links: string[][]) =>
+	links.map(([text, href]) => (
+		<Text key={text} component='a' href={href} fw={500}>
+			{text}
+		</Text>
+	))
+
+export const Footer = () => {
+	const { classes } = useStyles()
+
+	const support = makeLinks(supportLinks)
+
+	const connect = makeLinks(connectLinks)
+
+	return (
+		<Container className={classes.footer} fluid>
+			<BodyGrid style={{ padding: '40px' }} grow>
+				<Grid.Col md={5} sm={12}>
+					<Stack justify='space-between' style={{ height: '100%' }}>
+						<Stack align='start' spacing='xl'>
+							<Text>Inreach</Text>
+							<Title order={2} fw={500}>
+								{' '}
+								Seek LGBTQ+ resources. Reach safety. Find belonging
+							</Title>
+							<Button variant='primary' style={{ padding: '6px 48px' }} {...{ radius: 'md' }}>
+								Powered by Vercel
+							</Button>
+						</Stack>
+						<Text fz='xs'>InReach, Inc. 2023 • All rights reserved • InReach ❤️ Open Source</Text>
```

Comment:
Font size is too small - you can get the correct styling by adding a class to this using part of the theme file.

```ts
  copyrightText: {
    ...theme.other.utilityFonts.utility4,
    color: theme.other.colors.secondary.darkGray,
  }
```